File: TOKI/Problems/TOKI Open Contest Desember 2017/2A - Hari Pertama.py
# Wrong Answer

nums = list(map(int, input().split()))

x = nums[0] * nums[1]
div = x/9;

s = str('%.11f' % div)

s = s[:-10]

print(s, sep='', end='')
for i in range(10):
    print(s[len(s)-1], sep='', end='')

print()

File: TOKI/Problems/TOKI Open Contest Desember 2017/2A - Hari Pertama.cpp
// Wrong Answer

#include<bits/stdc++.h>
using namespace std;
typedef long long ll;

int main () {
    int a, b;
    ll x;
    double div;

    scanf("%d %d", &a, &b);
    
    x = (ll) a * b;
    div = (double) x / 9.0;

    string str = to_string(div);

    printf("%.11lf\n", div);

    return 0;
}